src/Controller/RequestsController.php

```diff
@@ -36,6 +36,8 @@ public function beforeFilter(Event $event)
         if (!Configure::read('debug')) {
             throw new NotFoundException();
         }
+
+        $this->response->header(['Content-Security-Policy' => "default-src 'self'; style-src 'self' fonts.googleapis.com; font-src 'self' fonts.gstatic.com"]);
```

Comment:
I didn't see an option to set a direct asset path, but I shouldn't be necessary, because everything on your own server should be ok to load. If the assets on your server are attacked .. well .. then you have more problems than just XSS attacks. :smile: 
I will look if there is an option to a direct asset path.